nullaway/src/test/java/com/uber/nullaway/jspecify/JSpecifyArrayTests.java

```diff
@@ -27,7 +27,7 @@
 import java.util.Arrays;
 import org.junit.Test;
 
-public class ArrayTests extends NullAwayTestsBase {
+public class JSpecifyArrayTests extends NullAwayTestsBase {
```

Comment:
Rename so we don't have two test classes named `ArrayTests` (for easier navigation in the IDE)